index.html

```diff
@@ -17,10 +18,126 @@
         <section class="main">
             <h1>Lekan Jimoh</h1>
             <p>Hello! I am a software developer! I can help you build a product, feature or website. Take a look of my works.
-            <br>If you like what you see and have a project you need coded, don’t hesitate and contact me.
+            <br>If you like what you see and have a project you need coded, don't hesitate and contact me.
             </p>
-            <button type="button"> Start collaboration</button>
+            <button class="get-cv" type="button"> Start collaboration</button>
         </section>
         <img class="background" src="img/logo1.svg" alt="layout">
+        <section class="container">
+            <section class="layout-2">
+                <h1>Projects</h1>
+                <img src="img/laptop.svg" alt="project1">
+                <div class="wrapper">
+                    <h2>Project Name goes there</h2>
+                    <div class="tech-button">
+                        <button class="item" type="button">HTML/CSS</button>
+                        <button class="item" type="button">Ruby on Rails</button>
+                        <button class="item" type="button">JavaScript</button>
+                    </div>
+                    <button class="project" type="button">see this project<i class="fa-solid fa-right-long"></i></button>
```

Comment:
- [ ] The project button design is a little different from the Figma template, Please fix the padding of the button and place some gap between the text and the arrow icon and make sure it matches the design on the template.

Your design

![Screenshot (87)](https://user-images.githubusercontent.com/83803414/153593034-05c92182-38ba-4406-862c-db7a8ab90d35.png)

Figma design

![Screenshot (88)](https://user-images.githubusercontent.com/83803414/153593060-a39ab426-38bd-4452-b144-089dfc08802e.png)